app/views/PDFView.scala.html

```diff
@@ -0,0 +1,122 @@
+                <h2 class="govuk-heading-m">@messages("pdf.logoTitle")</h2>
+            </div>
+            <div class="govuk-grid-column-two-thirds">
+                <h3 class="govuk-heading-m print-header__heading">
+                    @messages("pdf.previewTitle")
+                </h3>
+            </div>
+        </div>
+        <div class="grid-row">
+            <h3 class="govuk-heading-s govuk-!-static-margin-bottom-1">@messages("pdf.companyName")</h3>
+            <p class="govuk-body">SAMPLE LTD</p>
+        </div>
+        <div class="grid-row">
+            <h3 class="govuk-heading-s govuk-!-static-margin-bottom-1">@messages("pdf.utr")</h3>
+            <p class="govuk-body">123456</p>
+        </div>
+        <div class="grid-row print-banner">
+            @displayBanner(calculatorResult)._2
```

Comment:
It would be nice to return a case class here. _2 is not readable. Can you replace it with 

`case class Banner(title: String, html: Html)`

Reply:
Thats updated here and results page